bluesky/plans.py

```diff
@@ -3033,8 +2972,7 @@ def ramp_plan(go_plan,
 
         In seconds.
     '''
-    if md is None:
-        md = {}
+    md = {'plan_name': 'ramp_plan'}.update(md or {})
```

Comment:
The one line pattern here is pretty unsightly... Consider splitting it into two:

```python
md = {'plan_name': 'ramp_plan'}
md.update(md or {})
```